tests/e2e/actions.go

```diff
@@ -606,6 +605,11 @@ func (tr *TestConfig) startConsumerChain(
 	verbose bool,
 ) {
 	fmt.Println("Starting consumer chain ", action.ConsumerChain)
+
+	// Note that Soft Opt-Out has been deprecated. Nevertheless, the parameter still exists and is expected to be set
+	// because the SDK requires that all params are set to valid values in the genesis file.
+	action.GenesisChanges = action.GenesisChanges + ".app_state.ccvconsumer.params.soft_opt_out_threshold = \"0\""
```

Comment:
Why do you need to set it? You can delete this line.

Reply:
The default value is `""` which is a valid zero string.

Reply:
> Why do you need to set it? You can delete this line.

I cannot delete this line. If I do, I get the following error:
```
2024/06/21 11:02:14 getBalance() failed: /usr/local/bin/docker exec interchain-security-instance interchain-security-cd query bank balances consumer1sx6j9g2rh324a342a5f0rnx7me34r9nwduz5te --node tcp://7.7.8.253:26658 -o json: exit status 1
Error: post failed: Post "http://7.7.8.253:26658": dial tcp 7.7.8.253:26658: connect: no route to host
```
because the consumer chain never starts. Although we do not utilize `soft_opt_threshold`, the parameter has still to be set. This is why this line existed in all other E2E tests as well. In this PR, I just moved this line to a central place so we do not have to repeat it every time we create a new consumer chain.

> The default value is "" which is a valid zero string.

Yes, we could potentially set simply to `""` here but setting to `0` makes it clear that the soft opt-out feature is disabled.